bblfsh/client.py

```diff
@@ -1,54 +1,57 @@
 import os
-import sys
+import typing as t
 
 import grpc
 
-from bblfsh.aliases import (ParseRequest, ParseResponse, NativeParseRequest, NativeParseResponse,
-        VersionRequest, ProtocolServiceStub, SupportedLanguagesRequest, SupportedLanguagesResponse)
-from bblfsh.sdkversion import VERSION
-
-# The following two insertions fix the broken pb import paths
-sys.path.insert(0, os.path.join(os.path.dirname(__file__),
-                                "gopkg/in/bblfsh/sdk/%s/protocol" % VERSION))
-sys.path.insert(0, os.path.dirname(__file__))
+from bblfsh.aliases import (ParseRequest, DriverStub, ProtocolServiceStub,
+                            VersionRequest, SupportedLanguagesRequest, ModeType,
+                            VersionResponse)
+from bblfsh.result_context import ResultContext
 
 
 class NonUTF8ContentException(Exception):
     pass
 
 
-class BblfshClient(object):
+class BblfshClient:
     """
-    Babelfish gRPC client. Currently it is only capable of fetching UASTs.
+    Babelfish gRPC client.
     """
 
-    def __init__(self, endpoint: str):
+    def __init__(self, endpoint:str) -> None:
```

Comment:
(whitespace is needed before the type)
```suggestion
    def __init__(self, endpoint: str) -> None:
```